### DISCUSSION TITLE
How do I set Sloppiness by default to Architect in appState?

### DISCUSSION THREAD

**Discussion by pvaliyaveedan** [excalidraw/excalidraw#3909] [Q&A]
How to set the shape properties by default in the appState?

**Comment by dwelle** [comment]
Changing the properties should set them as default. That said, it resets when you clear the canvas. I was thinking we may change that.

**Reply by pvaliyaveedan** [reply]
So right now there is no way to change the sloppiness by default like currentItemfontFamily??

**Reply by dwelle** [reply]
It should work identically. Can you give more details on what's not working? What are the steps you're taking where the sloppiness doesn't persist while the other settings do? FYI, internally we call it `roughness`, and the appState property in question is named `currentItemRoughness`.

**Reply by pvaliyaveedan** [reply]
Yes! That worked. Thank you!
